src/apps/shared/commands/patches/m2_6879_create_deleted_respondents.py

```diff
@@ -0,0 +1,157 @@
+import os
+import uuid
+
+from sqlalchemy import func, select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Query
+
+from apps.answers.db.schemas import AnswerSchema
+from apps.workspaces.crud.user_applet_access import UserAppletAccessCRUD
+from apps.workspaces.db.schemas import UserAppletAccessSchema
+from apps.workspaces.domain.constants import Role
+from apps.workspaces.service.workspace import WorkspaceService
+from infrastructure.database import session_manager
+
+LOCAL_DB_PATCH_SQL = """
+        insert into user_applet_accesses (
```

Comment:
Something wrong with this query, it doesn't work properly.
Please replace with the next one or fix your's
```
insert into user_applet_accesses (
	id,
	is_deleted, 
	"role", 
	user_id, 
	applet_id, 
	owner_id, 
	invitor_id, 
	meta,
	created_at,
	updated_at,
	migrated_date 
)
select
	gen_random_uuid(),
	true,
	'respondent',
	answers.respondent_id,
	a.id,
	owner.user_id,
	owner.user_id,
	'{"secretUserId": "#deleted#"}'::jsonb,
	now() AT TIME ZONE 'UTC',
	now() AT TIME ZONE 'UTC',
	max(answers.migrated_date)
from applets a 
join user_applet_accesses owner on owner.applet_id = a.id and role = 'owner'
join answers on answers.applet_id = a.id
left join user_applet_accesses uaa on uaa.applet_id = answers.applet_id and uaa.user_id = answers.respondent_id
where 1=1
	and uaa.id is null
group by a.id, answers.respondent_id, owner.user_id
```

Reply:
probably the issue with distinct ..., migrated_date